Core/Graphics/Source/GraphicsImpl.cpp

```diff
@@ -45,14 +45,22 @@ namespace Babylon
         return static_cast<WindowType>(m_state.Bgfx.InitState.platformData.nwh);
     }
 
-    void GraphicsImpl::SetNativeWindow(const GraphicsConfiguration& config)
+    void GraphicsImpl::UpdateWindow(const WindowConfiguration& config)
     {
         std::scoped_lock lock{m_state.Mutex};
         m_state.Bgfx.Dirty = true;
         ConfigureBgfxPlatformData(config, m_state.Bgfx.InitState.platformData);
         UpdateDevicePixelRatio();
     }
 
+    void GraphicsImpl::UpdateContext(const ContextConfiguration& config)
```

Comment:
Didn't you say this can't be called a second time? If that's the case, then it seems like we should remove the `UpdateContext` function and just put this logic directly in `CreateGraphics(const ContextConfiguration& config)`.

Reply:
Between createGraphics being static and m_impl, I don't think I can put the code in there.